### PR TITLE
Node.js: dev/data-modeling/data-types

### PR DESCRIPTION
**Pull request by lukebakken**
Full, working example is here:

https://github.com/basho/riak-nodejs-client-examples/tree/master/dev/data-modeling

### CODE REVIEW

File: source/languages/en/riak/dev/data-modeling/data-types.md [hunk 1]
```diff
@@ -1125,3 +1324,12 @@ bruce.as_json()
...
 
+```javascript
+/*
+ * Since the User model object does not contain Riak Data Type
+ * information internally, Javascript can serialize the object
+ * directly
+ */
+JSON.stringify(joe);
+```
+
```
Comment (laurenrother) [inline source/languages/en/riak/dev/data-modeling/data-types.md]: Does this last space need to be here for the example to render? I didn't think it did, but I could be wrong.

Comment (lukebakken) [inline source/languages/en/riak/dev/data-modeling/data-types.md]: Nope! Just an extra line.

**Comment by lukebakken** [comment]
@laurenrother all set

**Comment by laurenrother** [comment]
:cake:

**Pull request merged**